iml-services/iml-device/Cargo.toml

```diff
@@ -18,8 +20,9 @@ tokio = { version = "0.2", features = ["macros"] }
 tokio-util = { version = "0.3", features = ["codec"] }
 tracing = "0.1"
 tracing-subscriber = "0.2"
-
 warp = "0.2"
+diesel = "1.4.4"
+tokio-diesel = { git = "https://github.com/jgrund/tokio-diesel" }
```

Comment:
This is re-exported by `iml-orm`.

Reply:
Fixed. The `diesel` itself stays for now, as `schema.rs` doesn't work with `pub use` in `iml-orm`.